tket/src/Mapping/LexiRoute.cpp

```diff
@@ -510,8 +508,26 @@ LexiRouteRoutingMethod::LexiRouteRoutingMethod(unsigned _max_depth)
     : max_depth_(_max_depth){};
 
 bool LexiRouteRoutingMethod::check_method(
-    const std::shared_ptr<MappingFrontier>& /*mapping_frontier*/,
-    const ArchitecturePtr& /*architecture*/) const {
+    const std::shared_ptr<MappingFrontier>& mapping_frontier,
+    const ArchitecturePtr& architecture) const {
+  std::set<Vertex> unplaced;
+  for (const std::pair<UnitID, VertPort>& pair :
+       mapping_frontier->quantum_boundary->get<TagKey>()) {
+    if ((mapping_frontier->circuit_.n_in_edges_of_type(
```

Comment:
Have you added this to the documentation of the lexi methods, that boxes are not longer valid input?
Maybe add a comment here in the code that makes clear why this check is done?